sql/driver/driver_test.go

```diff
@@ -149,6 +149,9 @@ func TestCreateTable(t *testing.T) {
 	if _, err := db.Exec("CREATE TABLE IF NOT EXISTS users " + cols); err != nil {
 		t.Fatal(err)
 	}
+	if _, err := db.Exec("CREATE TABLE employees " + cols); err != nil {
```

Comment:
I'm not clear on what this is testing. The previous `db.Exec` call is already testing that the session state stuff is working (otherwise `Session.database` would be empty). Perhaps I'm just missing something.